modules/internal/ChapelTaskData.chpl

```diff
@@ -28,10 +28,12 @@ module ChapelTaskData {
   // up to 16 bytes of wide pointer for _remoteEndCountType
   // 1 byte for serial_state
   // 1 byte for nextCoStmtSerial
+  // 1 byte for commDiagsTemporarilyDisabled
```

Comment:
`commDiagsTemporarilyDisabled` is quite wordy. For some reason I liked including `Temporarily`, but I'm not sure how much it adds. I'm open to different names if this seems too long to anybody.

Reply:
I don't think it's a name we need to be too picky about. I think the name you have has the advantage of not seeming to be the program-wide comms diagnostics switch.